core/ui/common/qam/powertools_menu.gd

```diff
@@ -0,0 +1,316 @@
+extends VBoxContainer
+
+var Platform := load("res://core/global/platform.tres")
+
+const AMD_GPU_MIN_MHZ: float = 200
+const POWERTOOLS_PATH : String = "/usr/share/opengamepadui/scripts/powertools"
+
+@onready var performance_manager := load("res://core/systems/performance/performance_manager.tres") as PerformanceManager
+@onready var launch_manager := load("res://core/global/launch_manager.tres") as LaunchManager
+
+var command_timer: Timer
+var update_timer: Timer
+
+@onready var cpu_boost_button := $CPUBoostButton
+@onready var cpu_cores_slider := $CPUCoresSlider
+@onready var gpu_freq_enable := $GPUFreqButton
+@onready var gpu_freq_max_slider := $GPUFreqMaxSlider
+@onready var gpu_freq_min_slider := $GPUFreqMinSlider
+@onready var gpu_temp_slider := $GPUTempSlider
+@onready var power_profile_dropdown := $PowerProfileDropdown
+@onready var tdp_boost_slider := $TDPBoostSlider
+@onready var tdp_slider := $TDPSlider
+@onready var thermal_profile_dropdown := $ThermalProfileDropdown
+@onready var smt_button := $SMTButton
```

Comment:
There should be an `as <type>` added to these so the correct type is inferred. Right now they default to being inferred as a `Node` type.

Reply:
Done. I had to name our custom slider to do this. I called it ValueSlider. A second option was LabledSlider.